mongo/mongo.go

```diff
@@ -5,47 +5,77 @@ import (
 	"errors"
 	"strconv"
 
-	"github.com/globalsign/mgo"
-
-	"github.com/globalsign/mgo/bson"
-
+	"github.com/ONSdigital/dp-healthcheck/healthcheck"
+	dpMongoHealth "github.com/ONSdigital/dp-mongodb/v2/health"
+	dpMongoDriver "github.com/ONSdigital/dp-mongodb/v2/mongodb"
 	errs "github.com/ONSdigital/dp-recipe-api/apierrors"
 	"github.com/ONSdigital/dp-recipe-api/models"
 	"github.com/ONSdigital/log.go/log"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+const (
+	connectTimeoutInSeconds = 5
+	queryTimeoutInSeconds   = 15
 )
 
 // Mongo represents a simplistic MongoDB configuration.
 type Mongo struct {
-	Collection string
-	Database   string
-	Session    *mgo.Session
-	URI        string
+	Collection   string
+	Database     string
+	Connection   *dpMongoDriver.MongoConnection
+	Username     string
+	Password     string
```

Comment:
can you add the `JSON` omit tag to these sensitive fields please - just incase someone accidentally logs the object.